### PR TITLE
fix(setup-wallet): create wallet styling

### PR DESCRIPTION
**Pull request by banklesss**
##### Ticket

YOMO-1829

### CODE REVIEW

File: packages/theme/src/base-palettes/dark-palette.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const darkPalette: BasePalette = {
   primary_c400: '#2E4BB0',
   primary_c300: '#304489',
   primary_c200: '#142049',
-  primary_c100: '#171B28',
+  primary_c100: '#1F253B',
```
Comment (michalszorad) [inline packages/theme/src/base-palettes/dark-palette.ts:15]: Is this change confirmed by design team?

Comment (banklesss) [reply packages/theme/src/base-palettes/dark-palette.ts:15]: I got it from figma

Comment (jorbuedo) [reply packages/theme/src/base-palettes/dark-palette.ts:15]: Remember there's still this PR drafted with what we discussed in the meeting with design
https://github.com/Emurgo/yoroi/pull/3466/files#diff-e8c6e3f5ff65fa7eddec83cbae367693cbce191c2825deacc4d5cefa943bf51f

Comment (banklesss) [reply packages/theme/src/base-palettes/dark-palette.ts:15]: i hardcoded the values

Comment (banklesss) [reply packages/theme/src/base-palettes/dark-palette.ts:15]: and restored previous dark palette

Comment (banklesss) [reply packages/theme/src/base-palettes/dark-palette.ts:15]: there is something wrong on figma

image

**Review by michalszorad — APPROVED** [review]

File: packages/theme/src/base-palettes/dark-palette.ts (hunk 1 above)
Comment (banklesss) [reply packages/theme/src/base-palettes/dark-palette.ts:15]: changes confirmed by product

Comment (stackchain) [reply packages/theme/src/base-palettes/dark-palette.ts:15]: if it is needed now no problem we can deal with conflicts later, bringing it in

**Review by stackchain — APPROVED** [review]

**Pull request merged**